CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ## v0.20.0
 * Add option to disable MongoDB and RabbitMQ in-cluster deployment and configuration (#79)
 * Compose multi-node connection string for MongoDB cluster instead of using loadbalancer single host (#80)
+* Change etcd dependency from incubator/etcd to stable/etcd-operator (#81)
```

Comment:
`v0.20.0` was already released yesterday https://github.com/StackStorm/stackstorm-ha/releases/tag/v0.20.0

Let's tag it `v0.21.0` and also bump version in `Chart.yaml`